Nos/Models/CoreData/Relay+CoreDataClass.swift

```diff
@@ -45,6 +46,9 @@ public class Relay: NosManagedObject {
         ]
     }
     
+    // swiftline:disable:next force_unwrapping
```

Comment:
There's a typo here; I'm surprised this didn't fail SwiftLint... 🤔 

```suggestion
    // swiftlint:disable:next force_unwrapping
```

Do we not need it, maybe? Is SwiftLint not checking for force unwrapping?

Reply:
hmm I think it may be the lag in swiftlint output. I noticed that sometime I only see the warnings after a while. Also, my only feedback is seeing the yellow line appear. Is there a better place to see all the linter warnings in a single place?

Reply:
In Xcode you can see them all in the Issue navigator (command-5, the 5th tab in the left panel). You can also run swiftlint from the command line if you prefer.